dpgen2/entrypoint/args.py

```diff
@@ -78,6 +78,7 @@ def dp_train_args():
             "init_models_paths",
             list,
             optional=True,
+            default=None,
```

Comment:
It seems that `init_models_paths` cannot be set to `null` or `[]` if no init_model is available. I‘m not sure if it is in line with expectations.